vars/debian.yml

```diff
@@ -15,7 +15,7 @@ cvmfs_packages:
     - cvmfs
   stratum1:
     - apache2
-    - libapache2-mod-wsgi
+    - "{{ 'libapache2-mod-wsgi-py3' if ansible_distribution_release in ('bookworm', 'jammy','bullseye') else 'libapache2-mod-wsgi' }}"
```

Comment:
This will break for next distribution release that won't be declared in the list. If the current and future package must be `libapache2-mod-wsgi-py3`, then the condition must be set for `libapache2-mod-wsgi` imho.

Reply:
I reversed the control of the distributions for the support of future versions